client/etcd/options.go

```diff
@@ -35,14 +35,14 @@ type Options interface {
 	Zone() string
 	SetZone(z string) Options
 
-	ServiceInitTimeout() time.Duration
-	SetServiceInitTimeout(timeout time.Duration) Options
+	Service() string
```

Comment:
ServiceID usually means [this](https://github.com/m3db/m3cluster/blob/master/services/types.go#L134) in the kv world, so I'd rather not to confuse with it.